<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rebuild_win</class>
 <widget class="QDialog" name="rebuild_win">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>779</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>1200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Export</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Assets/Assets.qrc">
    <normaloff>:/assets/icons/icons8-compare-50.png</normaloff>:/assets/icons/icons8-compare-50.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	
	font: auto &quot;Arial&quot;;
	font-size: 14px;
font-weight:bold;

}




#rebuild_win{
	background-color: #ffffff;
	border: 5px solid rgb(77, 77, 77);
	border-radius: 10px;
	
}



QPushButton{
	border: none;
	min-height: 40px;
	border-radius: 5px;

	
}
/*
QPushButton:hover{

	background-color:rgb(22, 38, 76);

}

QPushButton:disabled{
	color: #808080;
	background-color:rgb(210, 210, 210);

}
*/

/*********************************************/
QLabel{
	color: rgb(50,50,50);
color: #404040;
}


/*********************************************/


/*******************************************/
QLineEdit
{
	border:2px solid rgb(6, 76, 130);
    border-radius: 3px;
    padding: 1px 18px 1px 8px;
    min-width: 6em;
	min-height: 35px;
	font-size: 16px;
}



/*******************************************/


QGroupBox{
	font-weight: bold;
}

QLineEdit:hover{
	border: 1px solid black;
	background: #e4f0fa;
}

QLineEdit:focus {
	color: rgb(6, 76, 130);
	background: #ffffff;
	selection-background-color: rgb(255, 204, 75);
}

QSpinBox:hover{
	background: #e4f0fa;
}


/*********************************************/
QComboBox
{
	border:2px solid rgb(6, 76, 130);
    border-radius: 3px;
    padding: 1px 18px 1px 8px;
    min-width: 6em;
	min-height: 35px;
	font-size: 14px;
}

QComboBox:enabled{
color: rgb(50, 50, 50);
}

QComboBox:disabled
{
	border:2px solid rgb(210, 210, 210);
	color:(210, 210, 210);

}

QComboBox::down-arrow
{   
	image: url(:/assets/Assets/icons/icons8-arrow-48.png);
	width: 15px;
    height: 15px;

}

QComboBox::drop-down
{
	 background-color: rgb(238, 134, 44);
	 min-width: 30px;
}

QComboBox::drop-down:disabled 
{
	 background-color: rgb(210, 210, 210);
	 min-width: 30px;
}

QComboBox QAbstractItemView {
    border: none;
    selection-background-color: rgb(6, 76, 130);
	selection-color: rgb(6, 76, 130);


}

QComboBox QAbstractItemView {
    border: none;
}

QComboBox QAbstractItemView::item {
    border: none;
	height:30px;
}


</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="close_btn">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	padding:3px;
	border:none;
margin: 3px;
}

QPushButton:hover{

border: 2px solid #404040;

}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="Assets/Assets.qrc">
         <normaloff>:/assets/icons/icons8-close-50.png</normaloff>:/assets/icons/icons8-close-50.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="leftMargin">
      <number>50</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>50</number>
     </property>
     <property name="bottomMargin">
      <number>20</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size:22px;
font-weight: bold;
color:rgb(12, 80, 139);</string>
       </property>
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignLeft">
      <widget class="QLabel" name="message_lbl">
       <property name="styleSheet">
        <string notr="true">margin: 10px 0px 0px 20px;</string>
       </property>
       <property name="text">
        <string>massage</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="wrong_codes_frame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="topMargin">
         <number>30</number>
        </property>
        <item>
         <widget class="QLabel" name="wrong_code_title">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>-1</pointsize>
            <weight>75</weight>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size:22px;
font-weight: bold;</string>
          </property>
          <property name="text">
           <string>invalid codes</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>number of undefined codes:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="wrong_count_lbl">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="wrong_codes_text_edit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>25</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="exception_frame">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="topMargin">
         <number>30</number>
        </property>
        <item>
         <widget class="QLabel" name="exception_title">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>-1</pointsize>
            <weight>75</weight>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size:22px;
font-weight: bold;</string>
          </property>
          <property name="text">
           <string>Exception</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="exception_error">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="ok_btn">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{

background-color: rgb(120, 120, 120);
color:#ffffff;
max-width: 200px;
}


QPushButton:hover{

background-color: rgb(80, 80, 80);
color:#ffffff;
}
</string>
         </property>
         <property name="text">
          <string>Ok</string>
         </property>
         <property name="shortcut">
          <string>Return</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Assets/Assets.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>ok_btn</sender>
   <signal>clicked()</signal>
   <receiver>rebuild_win</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>423</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>close_btn</sender>
   <signal>clicked(bool)</signal>
   <receiver>rebuild_win</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>679</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
